### PR TITLE
Aside Notification Centered Block Update

### PR DESCRIPTION
**Pull request by Sartxi**
The aside notification "centered" is authored to read the `data-align` attribute added by word when you set the text contents alignment. But when copying the aside from the sidekick library those attributes aren't being set. This update will make it so you can also set the centered text as a style variant in order to make the centered variation available when copying the block from the sidekick library.

Resolves: https://jira.corp.adobe.com/browse/MWPW-129469

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/sarchibeque/notification-centering?martech=off
- After: https://aside-notification-centered--milo--adobecom.hlx.page/drafts/sarchibeque/notification-centering?martech=off

### CODE REVIEW

File: libs/blocks/aside/aside.css [hunk 1]
```diff
@@ -352,11 +347,6 @@
   margin-bottom: var(--spacing-xxs);
 }
 
-.aside.notification.center.small .foreground.container .text,
-.aside.notification.center.small .foreground.container .text>* {
-  justify-content: flex-start;
-}
-
```
Comment (meganthecoder) [inline libs/blocks/aside/aside.css:359]: Why are you removing this? It doesn't seem related to data-align

**Review by ryanmparrish — APPROVED** [review]

**Review by ryanmparrish — APPROVED** [review]
